.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2870,7 +2870,15 @@ stages:
               --destination-path "$(Build.SourceVersion)" \
               --source "$(Build.ArtifactStagingDirectory)"
           displayName: Upload blobs to Azure
-          condition: and(succeeded(), eq(variables.isMainBranch, true))
+          condition: >
```

Comment:
This is the only actual change, the rest is just whitespace clean up from VS code.

I left the whitespace cleanup, as I thought it was probably useful.

Reply:
GitHub PR > ⚙️ > `Hide whitespace` 👌🏼

Reply:
Wow, didn't know about that, thanks!

Reply:
> This is github desktop only?

Web view 🙂 

![image](https://user-images.githubusercontent.com/18755388/212027812-4e667383-be68-43f7-ace3-2b7dbd46554a.png)